### ISSUE TITLE
unable to get "checkNode" working

### ISSUE THREAD

**Issue by budg1e** [hummingbird-dev/hummingbird-treeview#23]
hello -can anyone provide a working example? Just doesn't work for me. Have hijacked this: 

https://jsfiddle.net/z5amrc4x/1/

apologies if wrong

thanks in advance,
adrian

**Comment by hummingbird-dev** [comment]
Hi Adrian,
Sorry, there was an old version linked in the cdn. I've updated it now and the jsfiddle is working as expected.
Best regards,
Sebastian

**Comment by hummingbird-dev** [comment]
https://jsfiddle.net/m50yzkob/

**Comment by budg1e** [comment]
hello -thank you for getting back to me. I discovered the issue was with certain characters I was using, eg.

test1|test2

does not check an item

`<li data-id="test1|test2">-Goodfellas</li>`

Have updated that fiddle. Guessing it's a regex thing?

I've had to work around it using just IDs and filtering over to get these prior to setting. It's all ok now -just kicked me when I was in a hurry!

thanks again for getting back to me
